spec.bs

```diff
@@ -1139,6 +1184,8 @@ The output of running a FLEDGE generateBid() script, which needs to be scored by
   reportWin().
 : <dfn>modeling signals</dfn>
 :: Null or a {{unsigned short}}.
+: <dfn>interest group</dfn>
+:: An [=interest group=], whose generateBid() invocation generated this bid.

```

Comment:
```suggestion
:: An [=interest group=], whose `generateBid()` invocation generated this bid.

```

Reply:
I see there's another instance of generateBid in "ad cost" above - would be nice to wrap that in backticks as well for consistency.